test/e2e/models/CodeServer.ts

```diff
@@ -88,6 +88,13 @@ export class CodeServer {
       }),
       "utf8",
     )
+
+    const extensionsDir = path.join(__dirname, "../extensions")
+
+    await fs.writeFile(
+      path.join(dir, "languagepacks.json"),
+      `{"es":{"hash":"8d919a946475223861fa0c62665a4c50","extensions":[{"extensionIdentifier":{"id":"ms-ceintl.vscode-language-pack-es","uuid":"47e020a1-33db-4cc0-a1b4-42f97781749a"},"version":"1.70.0"}],"translations":{"vscode":"${extensionsDir}/ms-ceintl.vscode-language-pack-es-1.70.0/translations/main.i18n.json"},"label":"español"}}`,
```

Comment:
We could use `JSON.stringify` here for readability.

Reply:
That sounds like a better approach! I'll refactor